doc/02-spacegraphcats-use-cases.md

```diff
@@ -0,0 +1,307 @@
+# spacegraphcats: use cases and working with the output
+
+## Installing the spacegraphcats software and its dependencies
+
+Please see [Installing spacegraphcats](00-installing-spacegraphcats.md).
+
+## Spacegraphcats use cases
+
+### Metagenome bin completion
+
+As discussed in the [primer on metagenomics and assembly graphs](0a-primer.md), assembly and binning are lossy processes that leave many reads uncharacterized in metagenome analysis. 
+Spacegraphcats can be used to recover unassembled and unbinned reads that "belong" to a metagenome bin by using the metagenome bin as a query. 
+We recommend the [Atlas](https://metagenome-atlas.readthedocs.io/en/latest/) automated metagenome analysis pipeline to produce the initial bins. 
+(Alternatively, the [genome-grist](https://github.com/dib-lab/genome-grist) pipeline may be used to identify the reference genomes to in a metagenome that can be used as queries.)
```

Comment:
to identify the list of known reference genomes to be used as queries for a metagenome.